tests/integration/src/test/scala/mesosphere/marathon/integration/setup/MesosTest.scala

```diff
@@ -372,7 +377,7 @@ trait MesosClusterTest extends Suite with ZookeeperServerTest with MesosTest wit
     mesosMasterUrl,
     autoStart = false,
     config = mesosConfig,
-    waitForMesosTimeout = patienceConfig.timeout.toMillis.milliseconds
+    waitForMesosTimeout = patienceConfig.timeout.toMillis.milliseconds - 10.seconds
```

Comment:
Can you comment as to the reason for this value? Also, if patienceConfig.timeout is set low enough we could have a negative duration.

Reply:
what he said ^^